universum/analyzers/pylint.py

```diff
@@ -20,17 +20,10 @@ class PylintAnalyzer:
     @staticmethod
     def define_arguments():
         parser = argparse.ArgumentParser(description="Pylint analyzer")
-        parser.add_argument("--files", dest="file_list", nargs='+', required=True,
-                            help="Python files and Python packages for Pylint. "
-                                 "Files could be defined as a single python file"
-                                 " *.py or directories with __init__.py file in the directory.")
+        utils.add_files_argument(parser)
```

Comment:
In my opinion you can merge all three calls into one.

Reply:
you mean like utils.add_files_python_version_and_result_file(parser) ?
that seems a bit excessive to me